code/_onclick/click_override.dm

```diff
@@ -73,26 +75,31 @@
 		beam_from.Beam(target_atom, icon_state = "lightning[rand(1, 12)]", icon = 'icons/effects/effects.dmi', time = 6)
 		if(isliving(target_atom))
 			var/mob/living/L = target_atom
-			var/surplus_power = C.get_surplus()
+			var/powergrid = C.get_available_power() //We want available power, so the station being conservative doesn't mess with glove / dark bundle users
```

Comment:
I just realized that this is wrong, oops

Reply:
yeah just made the change